packages/scripts/src/upsertMetaDirective.ts

```diff
@@ -0,0 +1,33 @@
+// A CLI script that accepts a docs rST file path and an updated meta description
+// The script will update the meta description in the rST file in place
+// Usage: node build/upsertMetaDirective.js <path-to-rst-file> <meta-description>
+
+import path from "path";
+import fs from "fs";
+import { upsertMetaDirectiveInFile } from "./meta2rst";
+
+const args = process.argv.slice(2);
+
+const filePath = path.resolve(args[0]);
+if (!fs.existsSync(filePath)) {
+  console.error(`File does not exist: ${filePath}`);
+  process.exit(1);
+}
+
+const metaDescription = args[1];
+if (!metaDescription) {
+  console.error("Meta description is required");
+  process.exit(1);
+}
+
+try {
+  upsertMetaDirectiveInFile(filePath, {
+    description: metaDescription,
+    keywords: null,
```

Comment:
why always no keywords?

Reply:
Just because we don't do any generative stuff for that currently. Passing `null` here is a no-op as opposed to an overwrite - it'll leave any existing keywords alone.